pyhap/accessory.py

```diff
@@ -183,6 +183,9 @@ def pincode(self):
             self._pincode = util.generate_pincode()
         return self._pincode
 
+    def get_accessory_information(self):
```

Comment:
Could you please write the docs here? E.g. make it explicit that this method is specifically aimed at configuring the AccInfo Service (if that's your idea).